docs/guide/devops.md

```diff
@@ -1,96 +1,82 @@
 
 When running eZ Platform in the `dev` environment you have access to the standard Symfony Web Debug Toolbar. It is extended with some eZ Platform-specific information:
 
 ![eZ Platform info in Web Debug Toolbar](img/web_debug_toolbar.png "eZ Platform info in Web Debug Toolbar")
 
-##### SPI (persistence)
+#### SPI (persistence)
 
 This section provides the number of non-cached [SPI](repository.md#spi) calls and handlers. You can see details of these calls in the [Symfony Profiler](http://symfony.com/doc/current/profiler.html) page.
 
-##### SiteAccess
-
-Here you can see the name of the current SiteAccess and how it was matched. For reference see the [list of possible siteaccess matchers](siteaccess.md#available-matchers).
+#### SiteAccess
 
-## Configuration
+Here you can see the name of the current SiteAccess and how it was matched. For reference see the [list of possible SiteAccess matchers](siteaccess.md#available-matchers).
 
-### Logging and Debug Configuration
+## Logging and debug configuration
 
-#### Introduction
+Logging in eZ Platform consists of two parts.
+One are several debug systems that integrate with Symfony developer toolbar to give you detailed information about what is going on.
+The other is the standard [PSR-3](https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-3-logger-interface.md) logger, as provided by Symfony using [Monolog](https://github.com/Seldaek/monolog).
 
-Logging in eZ Platform consists of two parts, several debug systems that integrates with symfony developer toolbar to give you detailed information about what is going on. And standard [PSR-3](https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-3-logger-interface.md) logger, as provided by Symfony using [Monolog](https://github.com/Seldaek/monolog).
+### Debugging in dev environment
 
-#### Debugging in dev environment
+When using the Symfony `dev` [environment](environments.md), the system tracks additional metrics for you to be able to debug issues. They include [Stash](http://stash.tedivm.com/) cache use *(done by [StashBundle](https://github.com/tedivm/TedivmStashBundle))*, and [persistence cache](repository.md#persistence-cache-configuration) use.
 
-When using the Symfony dev [environment](environments.md), the system out of the box tracks additional metrics for you to be able to debug issues, this includes [Stash](http://stash.tedivm.com/) cache use *(done by [StashBundle](https://github.com/tedivm/TedivmStashBundle))*, and a [persistence cache](repository.md#persistence-cache-configuration) use.
-
-##### Reducing memory use
+#### Reducing memory use
 
 !!! tip
 
-    For long running scripts, instead head over to [Executing long-running console commands](../cookbook/executing_long_running_console_commands.md) for some much more relevant info.
+    For long-running scripts, see [Executing long-running console commands](../cookbook/executing_long_running_console_commands.md) for some much more relevant info.
```

Comment:
For more information on long-running scripts
remove:  for some much more relevant info